tests/catalog/test_models.py

```diff
@@ -266,17 +268,32 @@ def test_long_name(self):
         It may create problems for tag with long name.
         """
         name = 'Имя ' * 50
-        group = catalog_models.MockTagGroup.objects.first()
+        group = catalog_models.MockTagGroup.objects.create(name='Some group')
         try:
             tag = catalog_models.MockTag.objects.create(group=group, name=name)
             self.assertLessEqual(len(tag.slug), catalog.models.Tag.SLUG_MAX_LENGTH)
         except DataError as e:
             self.assertTrue(False, f'Tag has too long name. {e}')
 
+    # @todo #302:30m  Process more special symbols for slugs.
+    @unittest.expectedFailure
     def test_slugify_conflicts(self):
         slugs = [
             catalog_models.MockTag.objects.create(name=name).slug
-            for name in ['11 A', '1/1 A', '1 1 A']
+            for name in ['11 A', '1/1 A', '1 1 A', '1.1 A', '1-1 A', '1_1 A']
+        ]
+
+        self.assertEqual(len(slugs), len(set(slugs)), msg=slugs)
+
+    def test_slugs_for_cloned_tag_values(self):
```

Comment:
```suggestion
    def test_cloned_tag_varied_group_slugs(self):
```
Up to you

Reply:
i'll leave it because of few points:
- i want to leave `test_slugs_` at the beginning
- tag values are the same. It's not very common situation, so it's the corner case. That's the point of test problem
- y, groups varying, but it's a normal situation